mycroft/configuration/__init__.py

```diff
@@ -111,6 +111,10 @@ def load(config=None):
             try:
                 from mycroft.api import DeviceApi
                 setting = DeviceApi().find_setting()
+                location = DeviceApi().find_location()
+                if location:
+                    city = location.get("city")
+                    config["location"] = city["name"] + ", " + city["state"]["name"] + ", " + city["state"]["country"]["name"]
```

Comment:
We should keep the location in the config as it is, an obj. So, no need for this concat logic. Each skill will deal with the data as they need. Some willnuse only the city name, others some other data, others will do a concat etc.